plugins/make_plugin_rpm

```diff
@@ -218,7 +219,7 @@ EOF
 	else
 		(cd ${GIT_ROOT}; 
 			if [ -f requirements.sh ]; then
-				sh requirements.sh
+				sudo sh requirements.sh
```

Comment:
@MarkRiddoch Kindly confirm this change.

Reply:
Yes, happy with that. The least we run under sudo the better. In general we should only run what is really needed as root.